jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -1164,6 +1164,138 @@ ecma_builtin_global_object_encode_uri (ecma_value_t this_arg __attr_unused___, /
   return ecma_builtin_global_object_encode_uri_helper (uri, unescaped_uri_set);
 } /* ecma_builtin_global_object_encode_uri */
 
+/*
+ * Escape characters bitset:
+ *   One bit for each character between 0 - 127.
+ *   Bit is set if the character does not need to be converted to %xx form.
+ */
+static const uint8_t escape_set[16] =
+{
+  0x0, 0x0, 0x0, 0x0, 0x0, 0xec, 0xff, 0x3,
```

Comment:
Could you, please, clarify how ECMA-262 defined the values, i.e. how can the values be calculated?